src/schema/v2/me/collector_profile.ts

```diff
@@ -30,6 +30,10 @@ export const CollectorProfileFields: GraphQLFieldConfigMap<
   intents: { type: new GraphQLList(GraphQLString) },
   loyaltyApplicantAt: date,
   name: { type: GraphQLString },
+  privacy: {
+    type: GraphQLInt,
+    resolve: ({ privacy }) => privacy,
+  },
```

Comment:
nice 🙌
btw, you don't need `line 35` here since the field name and the property you're destructing are the the same :)